app/runner.go

```diff
@@ -114,6 +119,20 @@ func (m *MultiRunner) PrometheusCollectors() []prometheus.Collector {
 	return collectors
 }
 
+// HealthChecks implements HealthChecker
+func (m *MultiRunner) HealthChecks() []health.Check {
+	// TODO: there isn't a running property on the multirunner itself, should there be?
+	checks := []health.Check{}
+
+	for _, runner := range m.Runners {
+		if cast, ok := runner.(health.Checker); ok {
+			checks = append(checks, cast.HealthChecks()...)
+		}
```

Comment:
We should also check if `runner` implements `health.Check` (if it doesn't implement `heath.Checker`), and it it does, add it to `checks`